cli/compose/types/types.go

```diff
@@ -495,6 +495,8 @@ func (e External) MarshalJSON() ([]byte, error) {
 
 // CredentialSpecConfig for credential spec on Windows
 type CredentialSpecConfig struct {
+	// @TODO Config is not yet in use
+	Config   string `yaml:"-" json:"-"` // Config was added in API v1.40
```

Comment:
Wasn't sure what the best way was to allow me to update the vendor, but not yet "expose" this field, as there's still discussion about the current implementation in moby, so I wasn't sure if this part of the API would change because of that https://github.com/moby/moby/pull/38777#discussion_r259333961

/cc @ddebroy @dperny @wk8